tests/standalone/GiUsing.py

```diff
@@ -0,0 +1,18 @@
+# nuitka-skip-unless-imports: cairo
+
+import cairo
+import gi
+
+gi.require_version("Gtk", "3.0")
+gi.require_version("Gdk", "3.0")
+
+from gi.repository import Gdk, Gtk
+
+# isort:start
```

Comment:
It would appear that the isort start  moniker is something you wanted before the gi.repository import?